presto-docs/src/main/sphinx/connector/tpch.rst

```diff
@@ -51,7 +51,33 @@ Every TPCH schema provides the same set of tables. Some tables are
 identical in all schemas. Other tables vary based on the *scale factor*
 which is determined based on the schema name. For example, the schema
 ``sf1`` corresponds to scale factor ``1`` and the schema ``sf300``
-corresponds to scale factor ``300``. The TPCH connector provides an
-infinite number of schemas for any scale factor, not just the few common
-ones listed by ``SHOW SCHEMAS``. The ``tiny`` schema is an alias for scale
-factor ``0.01``, which is a very small data set useful for testing.
+corresponds to scale factor ``300``. The scale factor represents the approximate size,
+in bytes, of the entire set of tables when stored uncompressed. For example,
+``sf1`` implies that writing all tables to disk uncompressed would require approximately 1GB.
+The TPCH connector provides an infinite number of schemas
```

Comment:
This statement makes me wonder about how to generate data for any scale factor then. We should clarify that as well.

Reply:
@aditi-pandit That statement already exists in the documentation, I only added "which includes floating-point values", how would you suggest I clarify how to generate for any scale factor?

Reply:
@emily-chan : We will have to figure how to do that. We can address it in a subsequent PR.